chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/summarystatistics/email/SurveillanceStatisticsSectionCreator.java

```diff
@@ -3,6 +3,7 @@
 import java.util.List;
 
 import gov.healthit.chpl.dto.CertificationBodyDTO;
+import gov.healthit.chpl.scheduler.job.summarystatistics.StatisticsMassager;
```

Comment:
I see this imported in a bunch of files, but no changes to the files themselves. Is this import used?

Reply:
It was an existing class I moved to a different package.  It was previously in the same package as the files using it, so an import was not needed.